### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"

### DISCUSSION THREAD

**Discussion by ashuvax** [community/community#11801] [Copilot Conversations] [answered]
Hi Hello Recently I can not use with your plugin Write to me:
GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"
Can you help me?

**Comment by davecheney** [comment, marked as answer]
I’m sorry you are having difficulties. Is it possible that your employer, internet service provider, or government is intercepting your SSL traffic?

**Reply by Shlysz** [reply]
I have the same problem. I come from China. Is it because the Chinese government doesn't allow it?

**Reply by davecheney** [reply]
Sadly I think the GFW may be blocking your communication with GitHub's and Copilot's servers

**Reply by Shlysz** [reply]
Thank you.When I use another computer, it worked successfully. It may just be a matter of my settings.

**Reply by likesnow97** [reply]
> 我也有同样的问题。我来自中国。是因为中国政府不允许吗？

我也来自中国，我也有相同的问题，请问你解决了吗

**Reply by surqing** [reply]
> Thank you.When I use another computer, it worked successfully. It may just be a matter of my settings.

请问解决了吗，是哪个设置有问题呢

**Reply by 2499887806** [reply]
第二天发现又可以使用了,不知道为什么

**Comment by setLylToxlx** [comment]
@Sxlysz 请问下你是怎么处理的,我的也不行mac os

**Reply by Shlysz** [reply]
我的两个笔记本都是win，我那个报错只是因为我没有同意这个插件联网
还有我感觉我上一个梯子质量比较差，换了一个梯子就好了，我感觉和这个也有关系

**Reply by yushuifeng** [reply]
我也遇到同样的问题了。知道请求的地址是哪里吗？不知道能否通过加入白名单解决。

**Reply by yushuifeng** [reply]
找到了，网络地址不通
https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions

**Reply by duowb** [reply]
这个地址打开404了啊；可以fq，我还是这个错误。。

**Reply by tianqo** [reply]
相同问题，但是这个难道一定要用梯子连接吗？有没有可能和windows自带的网络配置有关？

**Reply by tianqo** [reply]
我的问题也解决了...只是因为一直开着steamcommunity这款加速器而导致IP错误，使得证书无法认证...

**Comment by surqing** [comment]
![image](https://user-images.githubusercontent.com/84219482/163801933-1f967584-4fb1-4c5b-8a86-b1334215d9dc.png)
是因为我之前还没有加入候选人名单吗？

**Comment by ZhangEliot** [comment]
在我的电脑上，我使用了steam++加速器，一直不行。然后把它关掉就好了。甚至不需要梯子都可以使用。
